File: Partie-Mobile/lib/models/exercice.dart
class Exercice {
  int? idExercice;
  String? nom;
  String? image;
  String? type;
  String? description;

  Exercice(
      {this.idExercice, this.nom, this.image, this.type, this.description});

  Exercice.fromJson(Map<String, dynamic> json) {
    idExercice = json['idExercice'];
    nom = json['nom'];
    image = json['image'];
    type = json['type'];
    description = json['description'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['idExercice'] = this.idExercice;
    data['nom'] = this.nom;
    data['image'] = this.image;
    data['type'] = this.type;
    data['description'] = this.description;
    return data;
  }
}

File: Partie-Backend/src/main/java/com/basketapp/basket_app/models/Exercice.java
package com.basketapp.basket_app.models;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Lob;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Exercice {

    @Id
    @GeneratedValue (strategy = GenerationType.IDENTITY)
     private  Long idExercice;
     private String nom;
     private String Description;
     private String Type;
     @Lob
     private String image;
}


